setup.cfg

```diff
@@ -81,3 +80,43 @@ console_scripts =
 # W504: line break after binary operator
 # F841: local variable '%s' is assigned to but never used
 ignore = E129,E265,E266,E402,E501,E741,I100,I101,W504,F841
+
+[coverage:run]
+source = ginga
+omit =
+    ginga/_astropy_init*
+    ginga/conftest*
+    ginga/cython_version*
+    ginga/setup_package*
+    ginga/*/setup_package*
+    ginga/*/*/setup_package*
+    ginga/tests/*
+    ginga/*/tests/*
+    ginga/*/*/tests/*
+    ginga/version*
+    */ginga/_astropy_init*
```

Comment:
Note: Tom R said this repetition is for `tox`. So might as well, in case you want to use `tox` in the future.